server/controllers/auth.js

```diff
@@ -46,6 +47,20 @@ exports.registerUser = asyncHandler(async (req, res, next) => {
         }
       }
     });
+    const shoppingList = {
+      name: "Shopping",
+      image:
+        "https://team-granny-smith-s3.s3.ca-central-1.amazonaws.com/default-images/Shopping.jpg",
+      userId: user._id
+    };
+    const wishlist = {
+      name: "Wishlist",
+      image:
+        "https://team-granny-smith-s3.s3.ca-central-1.amazonaws.com/default-images/Wishlist.jpg",
+      userId: user._id
+    };
+    await List.create(shoppingList);
+    await List.create(wishlist);
```

Comment:
You can pass an array to `List.create()` so that we only have to perform on instruction:
```suggestion
    await List.create([shoppingList, wishlist]);
```